test/conformance/testdata/conformance.yaml

```diff
@@ -2090,6 +2090,39 @@
     support running as UID / GID, or privilege escalation.'
   release: v1.15
   file: test/e2e/common/node/security_context.go
+- testname: Sysctls, prevent launching greylisted sysctl
+  codename: '[sig-node] Sysctls [LinuxOnly] [NodeFeature:Sysctls] should not launch
+    unsafe, but not explicitly enabled sysctls on the node [MinimumKubeletVersion:1.21]
+    [Conformance]'
+  description: 'Pod is created with a not specifically allowed sysctl. Pod must fail
+    to launch. [LinuxOnly]: This test is marked as LinuxOnly since Windows does not
+    support sysctls'
+  release: v1.21
+  file: test/e2e/common/node/sysctl.go
+- testname: Sysctls, reject invalid sysctls
+  codename: '[sig-node] Sysctls [LinuxOnly] [NodeFeature:Sysctls] should reject invalid
+    sysctls [MinimumKubeletVersion:1.21] [Conformance]'
+  description: 'Pod is created with one valid and two invalid sysctls. Pod should
+    not apply invalid sysctls. [LinuxOnly]: This test is marked as LinuxOnly since
+    Windows does not support sysctls'
+  release: v1.21
+  file: test/e2e/common/node/sysctl.go
+- testname: Sysctl, test sysctls
+  codename: '[sig-node] Sysctls [LinuxOnly] [NodeFeature:Sysctls] should support sysctls
+    [MinimumKubeletVersion:1.21] [Conformance]'
+  description: 'Pod is created with kernel.shm_rmid_forced sysctl. Kernel.shm_rmid_forced
+    must be set to 1 [LinuxOnly]: This test is marked as LinuxOnly since Windows does
+    not support sysctls'
+  release: v1.21
+  file: test/e2e/common/node/sysctl.go
+- testname: Sysctl, allow specified unsafe sysctls
+  codename: '[sig-node] Sysctls [LinuxOnly] [NodeFeature:Sysctls] should support unsafe
```

Comment:
this test appears to test the same functionality as the prior test since its using the same sysctl "kernel.shm_rmid_forced", is this an oversight?

Reply:
`kernel.shm_rmid_forced` is in the safe set.

The current two test cases code are totally the same for `Sysctl, test sysctls` and `Sysctl, allow specified unsafe sysctls`.

Reply:
This makes me question whether this test should be promoted then. No loss of coverage if we don't promote this?

Reply:
Holding approval until I see an answer here